tests/linalg_test.py

```diff
@@ -1679,5 +1680,62 @@ def testSchurBatching(self, shape, dtype):
       Ts, Ss = vmap(lax.linalg.schur)(args)
       self.assertAllClose(reconstruct(Ss, Ts), args, atol=1e-4)
 
+  @jtu.sample_product(
```

Comment:
These tests should probably go under `class ScipyLinalgTestCase` rather than `class LaxLinalgTestCase`

Reply:
moved.